test/functional/api/v1/versions_controller_test.rb

```diff
@@ -97,7 +97,7 @@ def self.should_respond_to(format)
       assert_response :success
       set_cache_header
 
-      Timecop.travel(Time.zone.now + 1) do
+      travel 1.second do
```

Comment:
@simi I didn't investigate it so much but https://github.com/rubygems/rubygems.org/commit/f87b848787d1a787d178f508ca22ef896ac8b3 may be answer to your question.